### PR TITLE
Feature/text input

### PR DESCRIPTION
**Pull request by vitkon**
This PR adds Input and Label components

### CODE REVIEW

File: demo/src/components/TextInputPlayground.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export default class InputPlayground extends Component {
+    render() {
+        return (
+            <div className={`playground playground--${playgroundName}`}>
+                <h2>{Input.displayName} component Playground</h2>
+                <Input
+                    name="username"
+                    className="username"
+                    data-one="one"
+                    //defaultValue="test123"
```
Comment (mattdell) [inline demo/src/components/TextInputPlayground.js]: Do we need this prop?

File: src/components/atoms/Input.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+Input.getClasses = (props) => {
+    const prefix = 'dd';
+    const block = `${prefix}-${Input.displayName.toLowerCase()}`;
+    const disabledMod = `${block}--disabled`;
+    const requiredMod = `${block}--required`;
+    const readonlyMod = `${block}--readonly`;
+    const typeMod = `${block}--${props.type}`;
+    const extraClasses = props.className.split(' ');
+
+    return cx(
```
Comment (mattdell) [inline src/components/atoms/Input.js]: Did not know there was a library for this :+1:

File: src/components/atoms/Input.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import React, { Component, PropTypes } from 'react';
+import _ from 'lodash';
+import cx from 'classnames';
+import isRequiredIf from 'react-proptype-conditional-require';
```
Comment (mattdell) [inline src/components/atoms/Input.js]: Also did not know about this library. Good stuff! :+1:

File: demo/src/components/TextInputPlayground.js (hunk 1 above)
Comment (vitkon) [inline demo/src/components/TextInputPlayground.js]: well spotted, i'll remove

**Pull request merged**